### ISSUE TITLE
Google image search: escape behavior in image preview should close preview

### ISSUE THREAD

**Issue by infokiller** [infokiller/web-search-navigator#161]
Really great that this is working as well now. But if I can suggest what I think would be a minor improvement:

When you highlight a picture and press ENTER, the preview pane opens on the right. Once the preview pane is closed again using ESC, instead of the previous thumbnail being selected (which would be preferable), the search text entry box is. 

I think it would also be nice if the selected thumbnail bold outline highlighting was a little thicker, so it's easier to see.

_Originally posted by @eggbean in https://github.com/infokiller/web-search-navigator/issues/133#issuecomment-647325554_

**Comment by infokiller** [comment]
@eggbean I made the outline thicker and added results navigation (which means you can configure the keybindings in the extension options as an alternative to Google's keybindings).
However, fixing the opening of the search box with escape will probably not go in to the next release and may take more time to fix. This issue will track progress.

As a workaround, you should be able to just leave the preview pane open (don't close it with escape) and keep navigating the results.

**Comment by eggbean** [comment]
Yes, I seem to have this update already, so I suppose it must be server based.

I have just noticed that once I remove ESC as the alternative shortcut to focus on the search box, this behaviour stops, and now the extension works pretty much perfectly as I would like 😃

**Comment by eggbean** [comment]
Actually. there are issues.  

Sometimes there are two types of highlighted thumbnail.  There is the black outlined one, from the extension, but when the preview pane is opened, the selected and highlighted thumbnail is a blue shaded box (preferable aesthetically). When the preview pane is closed with ESC, then both both highlighted boxes often remain, both independent of each other.

Also, it is not the last previewed thumbnail which is selected with the black outline - it's the first image instead (this is after disabling ESC as 'search box focus').

The only keys that work in the preview pane mode are left and right arrow, but I think this is okay, as the top right corner left/right icons are what I would like to be using with the keys anyway.

**Comment by infokiller** [comment]
Yes, this issue will be slightly complex to fix in a clean way. My suggested workaround is to always navigate with the extension keys (up/down and j/k by default), and if you want to navigate when the preview pane is open, navigate to the image you want with the extension keys (which will only move the black outline, the shaded blue with not move), then press space/enter to load it to the preview.
